src/main/java/graphql/execution/incremental/DeferredCallContext.java

```diff
@@ -18,6 +19,28 @@
 @Internal
 public class DeferredCallContext {
 
+    private final int startLevel;
+    private final int fields;
+
+    public DeferredCallContext(int startLevel, int fields) {
+        this.startLevel = startLevel;
+        this.fields = fields;
+    }
+
+    @VisibleForTesting
+    public DeferredCallContext() {
+        this.startLevel = 0;
+        this.fields = 0;
+    }
+
+    public int getStartLevel() {
+        return startLevel;
+    }
+
+    public int getFields() {
+        return fields;
+    }
+
     private final List<GraphQLError> errors = new CopyOnWriteArrayList<>();
```

Comment:
nit: move this before method declarations

Reply:
done